php/libraries/NDB_Factory.class.inc

```diff
@@ -120,6 +120,8 @@ class NDB_Factory
         }
 
         self::$config = $config;
+        // NDB_Config::singleton() is already loading the config file
+        // in a better way. this needs to be modified
```

Comment:
@driusan I was tempted to use this but it looks like it needs clean-up.

I'm planning to remove this comment before merge but let me know if you agree with it

Reply:
I don't understand it.